test/apps/arcgis/app.js

```diff
@@ -80,7 +80,7 @@ loadArcGISModules([
   });
 
   const extren = new ArcGISDeckExternalRenderer(sceneView, {
-    deckLayers: [
+    layers: [
```

Comment:
Seems like alse `ArcGISDeckExternalRenderer` is using `DeckProps`. So this should be `deck.layers` maybe?